Huawei/PIC-LX9/res/values/config.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_autoBrightnessResetAmbientLuxAfterWarmUp">true</bool>
+    <bool name="config_automatic_brightness_available">true</bool>
+    <integer name="config_autoBrightnessAmbientLightHorizon">10000</integer>
+    <integer name="config_autoBrightnessBrighteningLightDebounce">4000</integer>
+    <integer name="config_autoBrightnessDarkeningLightDebounce">8000</integer>
+    <integer name="config_autoBrightnessInitialLightSensorRate">-1</integer>
+    <integer name="config_autoBrightnessLightSensorRate">300</integer>
+    <integer name="config_brightness_ramp_rate_fast">200</integer>
+    <integer name="config_brightness_ramp_rate_slow">40</integer>
+    <integer-array name="config_autoBrightnessLcdBacklightValues">
+        <item>6</item>
+        <item>47</item>
+        <item>150</item>
+        <item>180</item>
+        <item>250</item>
+        <item>255</item>
+    </integer-array>
+    <integer-array name="config_autoBrightnessLevels">
+        <item>8</item>
+        <item>55</item>
+        <item>350</item>
+        <item>1600</item>
+        <item>2550</item>
+    </integer-array>
+    
+    <bool name="config_suspendWhenScreenOffDueToProximity">false</bool>
+    <bool name="config_supportSystemNavigationKeys">false</bool>
```

Comment:
Uh that's weird, all Huawei devices i know have it. You're sure of this?

This option is about opening notifications using fingerprint gestures.

Reply:
this values comes from /framework-res/res/values.
I test today with "true" : no change (opening notifications using fingerprint gestures no work)

Reply:
original /framework-res/res/values
[http://pix.toile-libre.org/upload/original/1577035521.png](url)

Reply:
You enabled fingerprint gestures in treble settings, huawei section?

Reply:
yes.
(my last edit) : phh Treble Settings/Huawei features/enable fingerprint gestures : active

Reply:
no work, so I remove this 2 options